services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.module.ts

```diff
@@ -1,8 +1,31 @@
 // This Module is a "leaf" module at the "outside" of the 121 Service Modules dependency tree. It should not deoend on any other 121 Service Modules.
+import { FinancialServiceProviderEntity } from '@121-service/src/financial-service-providers/financial-service-provider.entity';
+import { FinancialServiceProvidersModule } from '@121-service/src/financial-service-providers/financial-service-provider.module';
+import { ProgramFinancialServiceProviderConfigurationEntity } from '@121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configuration.entity';
+import { ProgramFspConfigurationController } from '@121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.controller';
+import { ProgramFinancialServiceProviderConfigurationsRepository } from '@121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.repository';
 import { ProgramFinancialServiceProviderConfigurationsService } from '@121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.service';
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 
 @Module({
-  providers: [ProgramFinancialServiceProviderConfigurationsService],
+  imports: [
+    TypeOrmModule.forFeature([
+      ProgramFinancialServiceProviderConfigurationEntity,
+      // TODO: This is needed because the repository thats used to get FSPs is the typeorm variant, not our custom one.
+      FinancialServiceProviderEntity,
+    ]),
+    // TODO: REFACTOR: This configuration module shouldn't depend on the FinancialServiceProviderModule. Try to refactor this out in the displayName item.
```

Comment:
I tried locally to remove FinancialServiceProvidersModule and it still compiles

Reply:
I immediately get an error and it doesn't compile so I'm inclined to leave it in:
![image](https://github.com/global-121/121-platform/assets/17876717/e85310cf-a7e2-4187-bf07-27610b72b4d7)